docs/faq/migration/token-migration.md

```diff
@@ -22,27 +22,28 @@ The token migration usually takes about 10-15min for the transaction to be final
 
 If you are holding LYXe within your own wallet, it will not be migrated automatically. You have to instantiate the migration process manually. There is likely an amount of LYXe that will never migrate due to lost wallets. Some people might even keep some LYXe for sentimental reasons.
 
-Exchanges holding LYXe might migrate the asset for their customers. For these announcements, please follow their official announcement channels.
+Exchanges holding LYXe might migrate the asset for their customers. However, this is not a process controlled by LUKSO directly. The following exchanges have performed an migration already:
 
-## What happens with LYXe being sent back to the LYXe Contract?
+- [KuCoin LYX Migration Announcement](https://www.kucoin.com/announcement/en-kucoin-has-completed-the-token-swap-of-lyxe-to-lyx-20230721)
+- [Gate.io LYX Migration Announcement](https://www.gate.io/article/33153)
 
-Any LYXe accidentally sent to the [LYXe Token address](https://etherscan.io/token/0xA8b919680258d369114910511cc87595aec0be6D) on Ethereum will be [automatically sent as LYX](https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5) to the sending address on the LUKSO Blockchain.
+For further announcements, please follow the exchange's official announcement channels.
 
-## Will exchanges migrate LYXe for their users?
+## What happens with LYXe being sent back to the LYXe Contract?
 
-Some exchanges will likely migrate LYXe for their users automatically. However, this is not a process controlled by LUKSO directly. Please follow their social media profiles for updates.
+Any LYXe accidentally sent to the [LYXe Token address](https://etherscan.io/token/0xA8b919680258d369114910511cc87595aec0be6D) on Ethereum will be [automatically sent as LYX](https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5) to the sending address on the LUKSO Blockchain.
```

Comment:
I'm not sure it is automatic. We will have to do it manually.

I mean like automatic in the sense that users don't have to do it but not automatic in the sense that it will happen instantly.

Reply:
I did not edit this section, seems to be mixed up in Git because I removed a heading. However will remove it and reframe the sentence 👍🏻 Good catch